src/v0/destinations/sendgrid/config.js

```diff
@@ -38,8 +38,9 @@ const TRACK_EXCLUSION_FIELDS = [
   "trackingSettings"
 ];
 
-const DELETE_CONTACTS_ENDPOINT = `${BASE_ENDPOINT}/marketing/contacts`;
+const DELETE_CONTACTS_ENDPOINT = `${BASE_ENDPOINT}/marketing/contacts?ids=[IDS]`;
 const MAX_BATCH_SIZE = 30000;
+const urlLimit = 8150;
```

Comment:
Is this the limit for the entire URL or just the `ids` query parameter?

Reply:
Only the ids.

Reply:
Then we should clarify that in the name. Currently, it is ambiguous.

Reply:
Done